includes/REST/StoreSettingControllerV2.php

```diff
@@ -68,6 +74,18 @@ public function register_routes() {
                     'methods'             => WP_REST_Server::READABLE,
                     'callback'            => [ $this, 'get_single_settings' ],
                     'permission_callback' => [ $this, 'get_settings_permission_callback' ],
+                    'args'                => [
+                        'group_id' => [
+                            'description' => __( 'Unique identifier for the settings group.', 'dokan' ),
```

Comment:
I think that it is not *Unique identifier*

Reply:
Bhai, based on this method, I thought it's a unique identifier. Here, the group existence depended on `group_id`. 

```
    public function get_settings_group( string $group_id ) {
        $group_exist = $this->search_group( $group_id );

        if ( is_wp_error( $group_exist ) ) {
            return $group_exist;
        }

        $settings = apply_filters( 'dokan_vendor_rest_settings_for' . $group_id . '_page', [] );

        return array_map( [ $this, 'populate_settings_elements' ], $settings );
    }
```